src/main/resources/swagger/api-docs.yaml

```diff
@@ -1559,6 +1559,125 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+  /api/azure/v1/actionManagedIdentity/{billingProfileId}/{resourceTypeName}/{resourceId}/{action}:
+    post:
+      tags:
+        - Azure
+      summary: creates an action managed identity that the calling user has access to
+      operationId: createActionManagedIdentity
+      parameters:
+        - name: billingProfileId
```

Comment:
Just noticing: the [getPetManagedIdentity](https://sam.dsde-prod.broadinstitute.org/#/Azure/getPetManagedIdentity) endpoint takes the MRG coordinates in the POST body, and looks up the billing profile ID from the Sam DB. This endpoint does the opposite: it takes the billing profile ID in the path and looks up the MRG coordinates from the Sam DB.

I guess either style works because Sam stores a mapping between billing profile id <> MRG coordinates. Do we want to be consistent?

Reply:
We want to be consistent, and I actually want to migrate Pet Azure Managed Identities to just use the Billing Profile ID in the path. But, that would be a separate PR.

Reply:
Sounds good!